spring-application/library-borrow-service/src/main/resources/application-dev.yml

```diff
@@ -24,9 +25,35 @@ logging:
     org:
       springframework:
         security: debug
+
 server:
   error:
     include-stacktrace: never
   port: 9091
   servlet:
     context-path: /library-borrow
+
+library:
```

Comment:
The borrow-service will be independent of the library- and user-services, as the integration of all these services is managed by the library-facade-server (see my branch use_new_webclient_insteadof_deprected_template_approach).